config/deploy.rb

```diff
@@ -8,7 +8,7 @@
 # ask :branch, proc { `git rev-parse --abbrev-ref HEAD`.chomp }
 
 # Default deploy_to directory is /var/www/my_app
-set :deploy_to, '/apps/sidewalks'
+set :deploy_to, '/home/deploy/sidewalks'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.